app/models/user.rb

```diff
@@ -13,4 +13,8 @@ class User < ApplicationRecord
   def latest_three_posts
     posts.order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
- kudos  that you create  user.admin? method inside the user model 🔥💪🏻✅🎉

Reply:
@Reem-lab thanks a lot yeah I really worked on myself 😅😅

Reply:
Thanks